lldb/source/Interpreter/OptionArgParser.cpp

```diff
@@ -262,8 +262,10 @@ OptionArgParser::DoToAddress(const ExecutionContext *exe_ctx, llvm::StringRef s,
   // 3: The symbol/reg name if there is an offset
   // 4: +/-
   // 5: The offset value.
+  // clang-format off
   static RegularExpression g_symbol_plus_offset_regex(
-      "^(\\$[^ +-]+)|(([^ +-]+)([-\\+])[[:space:]]*(0x[0-9A-Fa-f]+|[0-9]+)[[:space:]]*)$");
+      "^(\\$[^ +-]+)|(([^ +-]+)[[:space:]]*([-\\+])[[:space:]]*(0x[0-9A-Fa-f]+|[0-9]+)[[:space:]]*)$");
```

Comment:
this is using ` ` (in `[^ +-]`)and `:space:` in the same regex, which seems oddly inconsistent?
Apart from Pavel's comment, this seems fine.

Reply:
Yea I think the regex can be simplified. But it's was using `:space:` already and I didn't want to make drive-by changes to this regex in this PR

Reply:
I'd probably prefer to just parse the string if we did want to simplify this logic